.github/workflows/README.md

```diff
@@ -0,0 +1,15 @@
+There are two GitHub Action pipelines for [Continuous Integration(CI)](ci.yml) and [Continuous Deployment(CD)](cd-training-pipeline.yml) of the ML pipeline on Vertex AI environment. 
+
+**CI**: This is to verify the proposed changes of the pipeline by creating and compiling through [TFX CLI](https://www.tensorflow.org/tfx/guide/cli). Specifically, `tfx pipeline create` and `tfx pipeline compile` returns errors if the codes of the pipeline has import/syntax problems. 
+- You can furthure improve CI pipeline by actually run the pipeline locally with `tfx run create`. However, in this case, you need to consider the code structure to be able to set epochs and datasets differently. You certainly don't want to run the pipeline over the long epochs and full dataset since the purpose of CI is to check if there is no problem.
```

Comment:
```suggestion
You can also _run_ the pipeline locally with `tfx run create`. However, in this case, you need to consider the code structure to be able to set the number of epochs and datasets differently. You certainly don't want to run the pipeline over for a larger number of epochs and on the full dataset since the purpose of CI is to check if there is no problem.
```